x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/components/agent_policy_advanced_fields/index.test.tsx

```diff
@@ -118,5 +128,26 @@ describe('Agent policy advanced options content', () => {
       });
       expect(mockUpdateAgentPolicy).toHaveBeenCalledWith({ is_protected: true });
     });
+    describe('when the defend integration is not installed', () => {
```

Comment:
Do we have test cases for when it has Defend Integration? If not, can we add those?

Reply:
yes, the other tests above have the defend integration inside the package policy